src/screens/Settings.js

```diff
@@ -48,26 +50,26 @@ export class Settings extends React.Component {
       marginBottom: 24,
     },
     networkView: {
-      backgroundColor: 'rgba(227, 0, 82, 0.1)',
+      backgroundColor: 'rgba(140, 70, 255, 0.1)',
```

Comment:
Again, the same color here.

Reply:
Created a method in constants to get this color depending on the alpha value.